ruby/red-arrow/test/test-record-batch.rb

```diff
@@ -54,4 +54,9 @@ class RecordBatchTest < Test::Unit::TestCase
                    records.collect {|record, i| [record.index, i]})
     end
   end
+
+  test("#to_table") do
+    assert_equal(Arrow::Table.new(@schema, [@counts]),
```

Comment:
It seems necessary to use 3.3.1 version of Ruby/GObjectIntrospection to pass an array of GObject.
CI is using 3.3.0.
https://travis-ci.org/apache/arrow/jobs/476565787#L6837

Reply:
Good catch!
I've released gobject-introspection gem 3.3.1.
I changed to require gobject-introspection gem 3.3.1 or later.